elasticsearch/src/main/java/org/apache/calcite/adapter/elasticsearch/PredicateAnalyzer.java

```diff
@@ -148,6 +148,7 @@ private static boolean supportedRexCall(RexCall call) {
         case OR:
         case LIKE:
         case EQUALS:
+        case IN:
```

Comment:
If I remember well the `IN` operator has been superseded by `SEARCH` so this code may be unreachable at the moment.